packages/google_maps_flutter/google_maps_flutter_android/pigeons/messages.dart

```diff
@@ -13,10 +13,10 @@ import 'package:pigeon/pigeon.dart';
 
 /// Pigeon equivalent of LatLng.
 class PlatformLatLng {
-  PlatformLatLng({required this.lat, required this.lng});
+  PlatformLatLng({required this.latitude, required this.longitude});
 
-  final double lat;
-  final double lng;
+  final double latitude;
```

Comment:
if you know can you add some more information here about what the units are for latitude/longitude?

Reply:
I'm not sure offhand what the units are; I assume degrees? In general I haven't been commenting the Pigeon data objects that correspond directly to some other class within the plugin.